frontend/src/views/groups/GroupsView.tsx

```diff
@@ -12,27 +12,40 @@ export const GroupsView = () => {
   const { setUser } = useContext(UserContext);
   const navigate = useNavigate();
 
-  const params = useParams<{ groupId?: string }>();
-  const selectedGroupId = parseInt(params.groupId ?? "") || undefined;
+  const params = useParams<{ groupName?: string }>();
+  const selectedGroupName = params.groupName;
 
   const { data: user } = useQuery({
     queryKey: ["groupsData"],
     queryFn: () =>
       axios.get(ME_URL).then((res: AxiosResponse<User>) => {
-        setUser({ user_id: res.data.user_id });
-        return res.data;
+        const user = res.data;
+        user.groups.sort((a, b) => a.name_short.localeCompare(b.name_short));
+        const duplicateNames = new Map<string, number>();
```

Comment:
I don't really see a reason why to include this part of the code anymore. There shouldn't ever be a duplicate `name_short` coming from the backend after setting the table field to unique.

Reply:
Thought i removed it. will fix